src/components/Search/SearchRouter/SearchButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import {PressableWithoutFeedback} from '@components/Pressable';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Permissions from '@libs/Permissions';
+import {useSearchRouterContext} from './SearchRouterContext';
+
+function SearchButton() {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {openSearchRouter} = useSearchRouterContext();
+
+    if (!Permissions.canUseNewSearchRouter()) {
+        return;
+    }
+
+    return (
+        <PressableWithoutFeedback
+            accessibilityLabel=""
+            style={[styles.flexRow, styles.mr2, styles.touchableButtonImage]}
+            onPress={() => {
```

Comment:
Not blurring the search button causes quite a unique edge case in mweb android platform that resulted in blue frame on the search button as mentioned in #52128